File: Assignment26/Program2.c


//Problem statement :Write a program which accept String from user and count number of small character 
//
//Input : 9
//Output: "Marvellous"

#include<stdio.h>

/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//             Function Name          :    CountSmall
//             Description            :    Function to count small characters in letters 
//             Intput Argument        :    char
//             Output Argument        :    char
//             Author                 :    Swati Vinayak Lahane 
//             Date                   :    19 November 2023
//             
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

int CountSmall(char *Str)
{
    int icnt = 0;
     
         
    
    while(*Str != '\0')
    {
        if((*Str >= 'a') && (*Str <= 'z'))
        {
            icnt++;
        }
        *Str++;
    }
  return icnt;
}


int main()
{
    char arr[20];
    int iRet = 0;

    printf("Enter String \n");
    scanf("%[^'\n']s",arr);

    iRet = CountSmall(arr);

    printf("Frequency of small letter: %d",iRet);

    return 0;
}

File: Assignment32/Program2.java
import java.util.*;

//Problem statement
//2 .Write a java Program Which accept string from from user and count number of small character 


//Input : Marvellous
//Output : 9

class Program2
{   

   public static int CountSmall(String str)
   {
       int icnt =0;
       int iCount = 0;
       char Arr[] = str.toCharArray();

       for(icnt =0; icnt < Arr.length; icnt++)
       {
           if((Arr[icnt] >= 'a') && (Arr[icnt] <= 'z'))
           {
               iCount++;
           }
       }

        return iCount;
   }
   public static void main(String Args[])

   {
      Scanner sobj = new Scanner(System.in);
      String str =null;
       int iRet = 0; 

      System.out.println("Enter The String:");
      str = sobj.nextLine();
    
      iRet = CountSmall(str);
      System.out.println("Count of small characters are :"+iRet);

      sobj.close();
   
   }
}